.github/workflows/ci.yaml

```diff
@@ -213,7 +213,7 @@ jobs:
         python:
           - '3.8'
           - '3.9'
-          # XXX TODO: Add 3.10 here once supported by Conda/Bioconda/Conda Forge.
+          - '3.10'
```

Comment:
The commit message for this change is titled:

>  CI: Enable Python 3.10 testing of Conda package 

But there's no testing of a Conda package here. The reason Conda is implicated in the removed comment is because the `./src/.github/actions/setup-integration-tests` action used in this job relies on Conda to set up the test environment. But the actual package the job tests is a Python package, not a Conda package.

Reply:
Oh, I remember being confused - that makes a lot more sense now. Updated the message in ea0b76674d4ce0bc30201b18d9928283a67e0139